### ISSUE TITLE
Bug report: Look-behind assertion produces syntax error in Find / Replace operation

### ISSUE THREAD

**Issue by jgmarcel** [gchq/CyberChef#936]
**Describe the bug**
A previously-working recipe (as of 22Oct19) now gets me an error.

**To Reproduce**
Steps to reproduce the behaviour or a link to the recipe / input used to cause the bug:

1. Go to `https://gchq.github.io/CyberChef/`
2. Click on 'Load recipe'
3. Paste the recipe below to the 'Recipe' field:
```
Remove_whitespace(false,true,false,true,true,false)
Find_/_Replace({'option':'Regex','string':'([^\\n]*)(?<=\\S)\\s*'},'"$1", ',true,false,true,false)
Find_/_Replace({'option':'Regex','string':'^(.*), $'},'[$1]',true,false,true,false)
```
4. See error in the 'Output' field:
`Find / Replace - SyntaxError in https://gchq.github.io/CyberChef/modules/Regex.js on line 36.<br><br>Message: invalid regexp group`

**Expected behaviour**
The recipe should work, cleaning and formatting input copied from Microsoft Excel into a valid Python list.

* Version: 9.12.0
* Compile time: 20/12/2019 16:10:42 UTC
* User-Agent:
Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:72.0) Gecko/20100101 Firefox/72.0
* [Link to reproduce](https://gchq.github.io/CyberChef/#recipe=Remove_whitespace(false,true,false,true,true,false)Find_/_Replace(%7B'option':'Regex','string':'(%5B%5E%5C%5Cn%5D*)(?%3C%3D%5C%5CS)%5C%5Cs*'%7D,'%22$1%22,%20',true,false,true,false)Find_/_Replace(%7B'option':'Regex','string':'%5E(.*),%20$'%7D,'%5B$1%5D',true,false,true,false)&input=null)

**Additional context**
I believe the syntax error is being caused by the look-behind assertion — `(?<=\S)` — in the first 'Find / Replace' operation. It used to be supported, but apparently is not anymore.

**Comment by n1474335** [comment]
Look-behinds have never been supported by JavaScript, so I'm not sure why this worked for you before... Are you sure the recipe hasn't changed since then?

**Comment by jgmarcel** [comment]
Thank you for the prompt reply.

While researching look-behind support by JavaScript, I found this: https://v8.dev/blog/regexp-lookbehind-assertions. Then I realized why it used to work before — I was using Google Chrome.

Just tested it again, using `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.117 Safari/537.36`, and confirm it works.

So is all JavaScript executed on client-side?
